### ISSUE TITLE
Role is unable to install without cypher-shell

### ISSUE THREAD

**Issue by Mehonoshin** [kevincoakley/ansible-role-neo4j#6]
Hey folks!

I've just tried to install this role, and it fails due to unmet dependency `cypher-shell` with version < 1.2.0.

I can create a PR to install this package from playbook, since package manager on ubuntu does not install it automatically.

WDYT?

**Comment by kevincoakley** [comment]
The issue appears to be with the Ubuntu neo4j package installing the latest version of cypher-shell (version 4.0.0 was released yesterday) instead of installing cypher-shell version 1.1.12 which is required by the 3.5.13 version of the neo4j package.  This isn't an issue on RedHat/CentOS.

I suspect this will be all cleared up shortly once neo4j 4.0.0 is released. 

Adding the following to the end of tasks/Debian.yml will fix the issue:

```
- name: Install cypher-shell 1.1.12
  package:
    name: cypher-shell=1.1.12
    state: present
```

Adding that fix would likely prevent neo4j 4.0.0 from installing properly once it is released, so I am not sure if I want to make that change to the master branch. I created the 3.5.13 branch at https://github.com/kevincoakley/ansible-role-neo4j/tree/3.5.13 which should contain the fix. Let me know if that doesn't work.
